wedlocks/src/templates/confirmEvent.js

```diff
@@ -0,0 +1,20 @@
+export default {
+  subject: () => 'You are attending an event!',
+  text: params =>
+    `Hello,
+
+This is just a reminder that you are attending the event ${
+      params.eventName
+    } on ${params.eventDate}!
+
+When you're ask your ticket at the door, just click on the link there and open it using your crypto enabled web browser:
+
+${params.ticketLink}.
+
+You can also show the QR code attached to that email.
```

Comment:
```suggestion
You can also show the QR code attached to this email.
```
(GitHub didn't save this comment to my review! :( )